### PR TITLE
add configuration for python 3.7

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Documentation can be found [here](https://docs.microsoft.com/en-us/NimbusML/over
 
 `nimbusml` runs on Windows, Linux, and macOS. 
 
-`nimbusml` requires Python **2.7**, **3.5**, or **3.6**, 64 bit version only. Python 3.7 is not yet supported.
+`nimbusml` requires Python **2.7**, **3.5**, **3.6**, or **3.7**, 64 bit version only. Python 3.7 is not yet supported.
```
Comment (justinormont) [inline README.md]:
```suggestion
`nimbusml` requires Python **2.7**, **3.5**, **3.6**, or **3.7**, 64 bit version only. Python 3.8 is not yet supported.
```

File: src/python/docs/sphinx/make.bat [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @ECHO OFF
 
 pushd %~dp0
-set PYTHONINTERPRETER=%~dp0..\..\..\..\dependencies\Python3.6\python.exe
+set PYTHONINTERPRETER=%~dp0..\..\..\..\dependencies\Python7\python.exe
```
Comment (justinormont) [inline src/python/docs/sphinx/make.bat]: Should this be `Python3.7`?

File: README.md (hunk 1 above)
Comment (xadupre) [reply README.md]: Fixed by removing. 3.8 is not expected before the end of the year. numpy is not available for python 3.8.

File: src/python/docs/sphinx/make.bat (hunk 2 above)
Comment (xadupre) [reply src/python/docs/sphinx/make.bat]: Fixed.

Comment (ganik) [reply src/python/docs/sphinx/make.bat]: no need for this. docs can be generated by python 3.6

---
In reply to: [272979237](https://github.com/Microsoft/NimbusML/pull/101#discussion_r272979237) [](ancestors = 272979237)

**Pull request merged**